### PR TITLE
don't disable repos while installing kernel

### PR DESCRIPTION
**Pull request by Torbuntu**
megi-kernel is a unique name and so now we don't need to disable repos to install the recommended software+megi-kernel. 
So it can all be done at once.

This was causing an error where pinephone-helpers could not find  iio-sensor-proxy from the main repo.

### CODE REVIEW

File: phone-scripts/02-install-packages.sh [hunk 1]
```diff
@@ -29,14 +29,11 @@ infecho "THIS WILL FAIL, DON'T WORRY ITS PROBABLY OK"
...
-dnf -y --disablerepo="*" --enablerepo="copr:copr.fedorainfracloud.org:njha:mobile" install megi-kernel
-
 infecho "Installing recommended packages..."
-dnf -y install feedbackd phoc phosh squeekboard gnome-shell ModemManager rtl8723cs-firmware \
+dnf -y install megi-kernel feedbackd phoc phosh squeekboard gnome-shell ModemManager rtl8723cs-firmware \
     f2fs-tools chatty calls carbons purple-mm-sms pinephone-helpers evolution-data-server \
-    f32-backgrounds-gnome kgx epiphany gnome-contacts evolution cheese NetworkManager-wwan \
-    lightdm-mobile-greeter firefox
+    f33-backgrounds-gnome kgx epiphany gnome-contacts evolution NetworkManager-wwan \
+    lightdm-mobile-greeter firefox mautilus
```
Comment (nikhiljha) [inline phone-scripts/02-install-packages.sh:36]: Did you mean `nautilus` here? Or is mautilius some kind of mobile nautilus or something :P

Comment (Torbuntu) [inline phone-scripts/02-install-packages.sh:36]: Adding nautilus. It doesn't come with an adaptive UI, but it does "sort" of work, and it is useful in Convergence mode.

File: phone-scripts/02-install-packages.sh [hunk 2]
```diff
@@ -29,14 +29,11 @@ infecho "THIS WILL FAIL, DON'T WORRY ITS PROBABLY OK"
...
 dnf -y install linux-firmware
 
-infecho "Installing kernel..."
-dnf -y --disablerepo="*" --enablerepo="copr:copr.fedorainfracloud.org:njha:mobile" install megi-kernel
-
 infecho "Installing recommended packages..."
-dnf -y install feedbackd phoc phosh squeekboard gnome-shell ModemManager rtl8723cs-firmware \
+dnf -y install megi-kernel feedbackd phoc phosh squeekboard gnome-shell ModemManager rtl8723cs-firmware \
     f2fs-tools chatty calls carbons purple-mm-sms pinephone-helpers evolution-data-server \
-    f32-backgrounds-gnome kgx epiphany gnome-contacts evolution cheese NetworkManager-wwan \
```
Comment (Torbuntu) [inline phone-scripts/02-install-packages.sh:38]: Grabbing f33 backgrounds and removing cheese, as it does nothing anyway on PinePhone/PineTab

File: phone-scripts/02-install-packages.sh (hunk 1 above)
Comment (Torbuntu) [reply phone-scripts/02-install-packages.sh]: Definitely meant `n` nautilus. Sorry about that.

**Comment by nikhiljha** [comment]
lgtm, but next time please use a more descriptive PR title

I usually squash+merge PRs so commit names are less important

**Pull request merged**